### PR TITLE
Competition

### PR DESCRIPTION
**Pull request by AknA13**

### CODE REVIEW

File: src/main/java/frc/robot/SwerveConstants.java [hunk 1]
```diff
@@ -126,15 +125,17 @@ public static final class OIConstants {
 	}
 
 	public static final class AutoConstants {
-		public static final double MAX_SPEED_METERS_PER_SECOND = 1; //3
-		public static final double MAX_ACCELERATION_METERS_PER_SECOND_SQUARED = 1; //3
+		public static final double MAX_SPEED_METERS_PER_SECOND = 2; //3
+		public static final double MAX_ACCELERATION_METERS_PER_SECOND_SQUARED = 2; //3
 		public static final double MAX_ANGULAR_SPEED_RADIANS_PER_SECOND = Math.PI / 2; // pi
 		public static final double MAX_ANGULAR_SPEED_RADIANS_PER_SECOND_SQUARED = Math.PI / 2; // pi
```
Comment (tsoim) [inline src/main/java/frc/robot/SwerveConstants.java]: the comments after each variable declaration is stale, please update

File: src/main/java/frc/robot/TeleopInputXBox.java
Comment (tsoim) [inline src/main/java/frc/robot/TeleopInputXBox.java:1]: this whole file is just commented out? please clean

File: src/main/java/frc/robot/systems/ElevatorWristFSM.java [hunk 2]
```diff
@@ -19,17 +23,22 @@ public class ElevatorWristFSM {
...
+	private static final double PID_CONSTANT_WRIST_P = 0.04;
 	private static final double PID_CONSTANT_WRIST_I = 0.00000001;
 	private static final double PID_CONSTANT_WRIST_D = 0.00000001;
 	private static final double OUTER_LIMIT_ENCODER = 100.0; //subject to change based on testing
-	private static final float MAX_UP_POWER = -0.1f;
-	private static final float MAX_DOWN_POWER = 0.1f;
+	private static final float MAX_UP_POWER = -0.15f;
+	private static final float MAX_DOWN_POWER = 0.15f;
 	private static final double WRIST_IN_ENCODER_ROTATIONS = 200; //16
 	private static final double WRIST_OUT_ENCODER_ROTATIONS = -200; //-40
```
Comment (tsoim) [inline src/main/java/frc/robot/systems/ElevatorWristFSM.java]: stale comments

File: src/main/java/frc/robot/systems/SpinningIntakeFSM.java
Comment (tsoim) [inline src/main/java/frc/robot/systems/SpinningIntakeFSM.java:1]: why is this removed? are we only using EveryBotIntakeFSM?

**Review by tsoim — COMMENTED** [review]
can you add a comment of what is enabled/disabled as part of this merge?

File: src/main/java/frc/robot/systems/SpinningIntakeFSM.java
Comment (AknA13) [reply src/main/java/frc/robot/systems/SpinningIntakeFSM.java:1]: yes, SpinningIntakeFSM was the FSM for last year's intake

**Comment by AknA13** [comment]
Competition changes:
-Increased autonomous drive distances and speeds
-Switched TeleopInput code from XBox controller to PS4 Controller
-Removed factory resetting code for SparkMaxs
-Fixed bug with intaking while flipped
-Added outtaking state for autonomous
-Enabled webcam feed
-Added double power states for wrist movement
-Added autonomous extension and retraction states for elevator
-Autonomous code has mid deposit and mobility

**Review by tsoim — APPROVED** [review]
